tests/protractor/e2e/sms-gateway.js

```diff
@@ -1,4 +1,6 @@
-const utils = require('../utils');
+const utils = require('../utils'),
+  commonElements = require('../page-objects/common/common.po.js'),
+  helper=require('../helper');
```

Comment:
Line up with the `utils` above, and add spaces around the `=` operator on line 3.